commands/issue.go

```diff
@@ -108,13 +108,18 @@ func createIssue(cmd *Command, args *Args) {
 }
 
 func writeIssueTitleAndBody(project *github.Project) (string, string, error) {
-	message := `
+	message := `%s
 # Creating issue for %s.
 #
 # Write a message for this issue. The first block of
 # text is the title and the rest is the description.
 `
-	message = fmt.Sprintf(message, project.Name)
+
+	if template := github.GetIssueTemplate(); template == "" {
+		message = fmt.Sprintf(message, "", project.Name)
```

Comment:
It seems that this `if/else` construct is unnecessary. The logic getting executed is identical regardless of the fact whether the `template` string is empty or not.